### ISSUE TITLE
Starting CDRS Manager Fatal null

### ISSUE THREAD

**Issue by liho00** [fonoster/routr#145]
<img width="1088" alt="image" src="https://user-images.githubusercontent.com/77238199/194841678-4014e2f4-35c1-4eb6-aede-04ef99f80fa4.png">

.env
```
# Example environment variables for running Routr on container environments
# including K8s.

USER_AGENT=Routr v1.0.0
IMAGE_VERSION=fonoster/routr:latest

DATA_SOURCE_PROVIDER=redis_data_provider
DATA_SOURCE_PARAMETERS=host=redis,port=6379

EXTERN_ADDR=192.168.1.108
LOCALNETS=
BIND_ADDR=127.0.0.1
RECORD_ROUTE=true
REGISTRAR_INTF=Internal

REST_SERVICE_PORT=4567
REST_SERVICE_BIND_ADDR=0.0.0.0
REST_SERVICE_MIN_THREADS=8
REST_SERVICE_MAX_THREADS=200
REST_SERVICE_TIMEOUT_MILLIS=5000
REST_SERVICE_UNSECURED=false
REST_SERVICE_KEY_STORE=etc/certs/api-cert.jks
REST_SERVICE_KEY_STORE_PASSWORD=changeit
REST_SERVICE_TRUST_STORE=etc/certs/api-cert.jks
REST_SERVICE_TRUST_STORE_PASSWORD=changeit

ACCESS_CONTROL_LIST_DENY=
ACCESS_CONTROL_LIST_ALLOW=
SECURITY_CONTEXT_KEY_STORE=etc/certs/domains-cert.jks
SECURITY_CONTEXT_KEY_STORE_PASSWORD=changeit
SECURITY_CONTEXT_TRUST_STORE=etc/certs/domains-cert.jks
SECURITY_CONTEXT_KEY_STORE_TYPE=jks
SECURITY_CONTEXT_CLIENT_AUTH_TYPE=DisabledAll
SECURITY_CONTEXT_CLIENT_PROTOCOLS=[TLSv1.2, TLSv1.1, TLSv1]
SECURITY_CONTEXT_DEBUGGING=false
CONFIG_FILE=config/config.yml

# SALT_FILE=/path/private/key
SALT=changeit

EX_RTP_ENGINE_ENABLED=false
EX_RTP_ENGINE_PROTO=http
EX_RTP_ENGINE_HOST=localhost
EX_RTP_ENGINE_PORT=8080

EX_UNIQUE_GATEWAY_PER_HOST_PORT=true
EX_CONVERT_TEL_TO_E164=false

SIP_TCP_PORT=5060
SIP_UDP_PORT=5060/udp
SIP_TLS_PORT=5061
SIP_WS_PORT=5062
SIP_WSS_PORT=5063

RTPE_PORT_MIN=30000
RTPE_PORT_MAX=30100

# Uncomment to enable remote JMX and remember to set the JMX port and external host
# JAVA_OPTS="-XX:NewRatio=2 -Xmx250M -Dcom.sun.management.jmxremote -Djava.rmi.server.hostname=192.168.1.3 -Dcom.sun.management.jmxremote.rmi.port=45856 -Dcom.sun.management.jmxremote.port=45856 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false"
```

docker-compose.yml
```
version: '3.7'

services:

  # SIP Server
  routr:
    image: ${IMAGE_VERSION}
    environment:
      - USER_AGENT
      - DATA_SOURCE_PROVIDER
      - DATA_SOURCE_PARAMETERS
      - EXTERN_ADDR
      - LOCALNETS
      - BIND_ADDR
      - RECORD_ROUTE
      - REGISTRAR_INTF
      - REST_SERVICE_BIND_ADDR
      - REST_SERVICE_MIN_THREADS
      - REST_SERVICE_MAX_THREADS
      - REST_SERVICE_TIMEOUT_MILLIS
      - REST_SERVICE_UNSECURED
      - REST_SERVICE_KEY_STORE
      - REST_SERVICE_KEY_STORE_PASSWORD
      - REST_SERVICE_TRUST_STORE
      - REST_SERVICE_TRUST_STORE_PASSWORD
      - ACCESS_CONTROL_LIST_DENY
      - ACCESS_CONTROL_LIST_ALLOW
      - SECURITY_CONTEXT_KEY_STORE
      - SECURITY_CONTEXT_KEY_STORE_PASSWORD
      - SECURITY_CONTEXT_TRUST_STORE
      - SECURITY_CONTEXT_KEY_STORE_TYPE
      - SECURITY_CONTEXT_CLIENT_AUTH_TYPE
      - SECURITY_CONTEXT_CLIENT_PROTOCOLS
      - SECURITY_CONTEXT_DEBUGGING
      - CONFIG_FILE
      - SALT
      - SALT_FILE
      - EX_RTP_ENGINE_ENABLED
      - EX_RTP_ENGINE_PROTO
      - EX_RTP_ENGINE_HOST
      - EX_RTP_ENGINE_PORT
      - EX_UNIQUE_GATEWAY_PER_HOST_PORT
      - EX_CONVERT_TEL_TO_E164
      - JAVA_OPTS
    ports:
      - '${REST_SERVICE_PORT}:4567'
      - '5060:5060'
      - '5060:5060/udp'
      - '5061:5061'
      - '5062:5062'
      - '5063:5063'
      # Uncomment to enable JMX monitoring (also see JAVA_OPTS at .env file)
      # - '45856:45856'
    volumes:
      - ./config/log4j2.yml:/opt/routr/config/log4j2.yml

  # Media relay engine
  rtpengine:
    image: psanders/rtpengine:latest
    restart: always
    # Network hostmode seems to only work in Linux systems
    network_mode: "host"
    environment:
      # This will fail if network_mode != host
      PUBLIC_IP: ${EXTERN_ADDR}
      PORT_MIN: ${RTPE_PORT_MIN}
      PORT_MAX: ${RTPE_PORT_MAX}
      LOG_LEVEL: '6'

  # Datasource
  redis:
    image: redis:6.0.3-alpine
    command: ['redis-server', '--appendonly', 'yes']
    hostname: redis
```

**Comment by psanders** [comment]
Hi @liho00 

Please try again with v1.0.6. We have a bug with v1.0.7 but a fix is on the way. 

The next patch version will fix this issue.

Sorry for the delayed response.

**Comment by psanders** [comment]
Found and fixed this issue. It will be available soon via `v1.0.8.`
